interface/wx/webview.h

```diff
@@ -297,6 +450,81 @@ class wxWebViewHandler
         @since 3.1.5
     */
     virtual wxString GetSecurityURL() const;
+
+    /**
+        When using the edge backend handler urls are https urls with a
+        virtual host. As default @c name.wxsite is used as the virtual hostname.
+        If you customize this host use a non existing site (ideally a reserved
```

Comment:
Minor too but helps readability a bit IMHO
```suggestion
        If you customize this host, use a non existing site (ideally a reserved
```